src/Compilers/Core/Portable/ConstantValue.cs

```diff
@@ -439,6 +439,7 @@ public object Value
                 {
                     case ConstantValueTypeDiscriminator.Bad: return null;
                     case ConstantValueTypeDiscriminator.Null: return null;
+                    case ConstantValueTypeDiscriminator.DefaultLiteral: return null;
```

Comment:
> I'm having some more trouble with constant values.

I don't think of ```default``` as a literal. It represent a default value of a type. Each type has a default value, but not all default values are constants from the language point of view. Let's talk offline in more details.